.idea/.idea.AirBelgie/Docker/compose.generated.override.yml

```diff
@@ -2,7 +2,7 @@
 services:
   airbelgie.web:
     build:
-      context: "/Users/rparkar/RiderProjects/AirBelgie"
+      context: "C:\\Users\\Rahul\\Documents\\GitHub\\backend"
```

Comment:
I feel like this file should be gitignore'd

Reply:
It probably should be. I need to figure out what's going on there and murder it off.

I'll do it in another PR